### PR TITLE
Add work and about section 

### PR DESCRIPTION
**Pull request by AbhishekkM05**
Added work and about section in my Portfolio project using html , CSS  and linters.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,8 +39,161 @@ <h1>I’m Abhishek <br>
...
          </div>
+</section>
 
 
-</section>   
+<section class="works">
+    <div class="project1">
+        <img src="assets/images/Tonic.png" class="img1" alt="">
+        <div class="task"><h2>Tonic</h2>
+            <div class="list">
```
Comment (jerryowusu) [inline index.html:49]: Kindly check and add the background color of `#E5E5E5` between the two sections. you can see the difference in the images show below
Figma design

Screenshot from 2022-09-23 18-56-39




Your design

Screenshot from 2022-09-23 18-56-53

Comment (AbhishekkM05) [inline index.html]: how to add that i tried ??

File: index.html [hunk 2]
```diff
@@ -39,8 +39,161 @@ <h1>I’m Abhishek <br>
        <!-- <li><a href="" class="link"><i class="material-icons">feedback</i></a> </li> -->
         </ul>
          </div>
+</section>
 
 
-</section>   
+<section class="works">
+    <div class="project1">
```
Comment (jerryowusu) [inline index.html]: Kindly check and add the background color of `#E5E5E5` between the two sections and in-between each the project cards. You can see the difference in the images shown below

Your design
Screenshot from 2022-09-23 19-02-17


Figma design
Screenshot from 2022-09-23 19-01-40

File: styles.css [hunk 3]
```diff
@@ -67,6 +71,226 @@ h1 {
...
+  font-weight: bold;
+  font-size: 16px;
+  margin: 4px 2px;
+  cursor: pointer;
+  margin-left: 50px;
+  background: #fff;
+  border-radius: 8px;
+  color: #396df2;
+  border: 1px solid blue;
+}
```
Comment (jerryowusu) [inline styles.css]: Please kindly implement the active, pressed and hover effects of all the buttons. It is true that these effects are not seen on mobile versions but it is a must to implement them at this stage of the project. (_check the last project bulleted point in the image below_).You can check the desktop side of the figma design for the button effects. :pray:

Screenshot from 2022-09-23 16-24-44

File: index.html [hunk 4]
```diff
@@ -39,8 +39,161 @@ <h1>I’m Abhishek <br>
...
+                    <li><img src="assets/images/Ellipse1-2.png" alt=""><a
+                            href="#" class="tech">CSS</a></li>
+                </ul>
+
+                <p class="skill">Frameworks</p>
+                <hr>
+                <p class="skill">Skills</p>
+               <hr>
+
+            </div>
```
Comment (jerryowusu) [inline index.html]: The languages, frameworks and skills should have the arrows as shown in the figma designs below. Kindly check and implement that. 

Screenshot from 2022-09-23 19-16-12

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @AbhishekkM05 , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses:

so-close

## Highlights
:heavy_check_mark: Good commit messages
:heavy_check_mark: Neat and well organized code
:heavy_check_mark: Excellent readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 3 above)
Comment (AbhishekkM05) [reply styles.css]: already done all except first part .

**Review by jerryowusu — APPROVED** [review]
Hi @AbhishekkM05 :wave: 

Well done for making the necessary changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

brilliant-well-done


## Highlights
:heavy_check_mark: Good commit messages
:heavy_check_mark: Neat and well organized code
:heavy_check_mark: Excellent readme file

## Optional suggestions

- [ ] As shown on the images below, there is a `bottom left border radius `
on the header section and a `top right border radius `on the about section which needs to be implemented. kindly check the images below and try as much as possible to implement them since it will make your project more beautiful and complete.

Screenshot from 2022-09-24 00-00-22
Screenshot from 2022-09-24 00-01-04

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**